doc/testing.md

```diff
@@ -135,10 +135,19 @@ the first N tiers they can afford to run, but at least tier1.
 
 A brief description of the tiered test groups:
 
-- `tier1`: This is the lowest test tier. Multiple developers run these tests
+- `tier1`: This is the lowest test tier.
```

Comment:
The use of "lowest" seems to imply an unstated directionality. Maybe there is better working, like "most basic".

Reply:
Changed to "most fundamental".